inspirehep/config.py

```diff
@@ -1623,3 +1623,10 @@
                'arxiv_eprints', 'public_notes', 'number_of_pages',
                'publication_info', 'earliest_date']
 }
+
+
+# App metrics
+# ===========
+APPMETRICS_ENABLED = False
```

Comment:
is this a feature flag? then it should follow the pattern and be something like `FEATURE_FLAG_ENABLE_APPMETRICS` and be defined close to the others.

Reply:
True that. Done.